PowerShellEditorServices.build.ps1

```diff
@@ -313,30 +313,55 @@ task RestorePsesModules -After Build {
     # Save each module in the modules.json file
     foreach ($moduleName in $moduleInfos.Keys)
     {
-        if (Test-Path -Path (Join-Path -Path $submodulePath -ChildPath $moduleName))
+        $moduleInstallPath = (Join-Path -Path $subModulePath -ChildPath $moduleName)
+
+        if (-not (Test-Path -Path $moduleInstallPath))
```

Comment:
I noticed that you changed the logic from:

```
if ( the path existed ) {
   continue
}

blah
blah
blah
```
to
```
if ( the path did not exist ) {
   blah
   blah
   blah
} else {
   it existed
}
```

The first organization is a bit cleaner and doesn't require an extra indent for a solid amount of code. Can you refactor it back?

Reply:
I can do

```
if ( the path existed ) {
   Write-Host... 
} else {
   Save module
}
```

But I want to keep the extra indent because I want to run the "move module out of subfolder" part regardless of wether or not the module already existed. Unless I can assume a clean build where the modules definitely aren't there, but then why check for them at all ;)

Reply:
Ah. I think that's a fair refactor then. :)